tooling/maven/camel-restdsl-openapi-plugin/pom.xml

```diff
@@ -55,6 +55,11 @@
             <artifactId>snakeyaml</artifactId>
             <version>${snakeyaml-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openapitools</groupId>
+            <artifactId>openapi-generator</artifactId>
+            <version>4.3.1</version>
```

Comment:
This can be a placeholder for property version in parent/pom

Reply:
Hi. Thanks for the input. Would you like the according property in the maven-plugins pom or in another, e.g. tooling-parent pom?

Reply:
Morning. I meant here: https://github.com/apache/camel/blob/master/parent/pom.xml

Reply:
Thanks